aws/lambda/python/s3_cleanup.py

```
@@ -0,0 +1,84 @@
+import boto3
+import time
+
+def lambda_handler(event, context):
+    bucket_name = event.get("bucket_name")
+
+    if not bucket_name:
+        return {"statusCode": 400, "body": json.dumps("Bucket name not provided")}
+
+    s3 = boto3.client("s3")
+    events = boto3.client("events")
+    lambda_client = boto3.client("lambda")
```

Ruff (0.8.2)

Local variable `lambda_client` is assigned to but never used

Remove assignment to unused variable `lambda_client`

(F841)